pkg/web/static/js/main.js

```diff
@@ -43,4 +43,26 @@ document.body.addEventListener('htmx:responseError', (e) => {
       document.querySelector('.alert').remove()
     }, 5000)
   }
-})
\ No newline at end of file
+})
+
+/*
+ * Timestamp handling: localizes and formats datetimes on the page.
+ */
+const updateDatetimes = (e) => {
+  const options = {};
+  const dtfmt = Intl.DateTimeFormat(navigator.languages, {
```

Comment:
Would we eventually want use a library like `dayjs` to handle this? We're currently using the CDN version to humanize `Last Update`. It has required installing additional plugins, but it could be easier to deal with once we switch to using the package.

Reply:
I have no problems switching to dayjs when we can start packing the full javascript -- heck maybe we should also use jQuery and underscore to make coding easier as well! If you think we should use dayjs right now, that's fine; we can close this PR and do that in a follow on.

Reply:
We can go ahead and use dayjs. I was thinking of waiting until we start using a bundler, but I can create a story to use dayjs now.

Reply:
Ok, if you're going to use dayjs - let's just go ahead and close this PR.